mailbox/api/src/main/java/org/apache/james/mailbox/MessageManager.java

```diff
@@ -117,6 +117,17 @@
      */
     Iterator<MessageUid> expunge(MessageRange set, MailboxSession mailboxSession) throws MailboxException;
 
+    /**
+     * Deletes messages
+     *
+     * @param uids
+     *            list of {@link MessageUid} to delete
+     * @param mailboxSession
+     *            not null
+     * @throws MailboxException
```

Comment:
When we write new javadoc we should only add information not only present in the method signature...